pkg/ingester/transfer.go

```diff
@@ -244,9 +251,8 @@ func (i *Ingester) TransferTSDB(stream client.Ingester_TransferTSDBServer) error
 			}
 			filesXfer++
```

Comment:
Unrelated from these changes, but shouldn't this counter incredemented only when a new file is created instead of each time a data chunk is transferred?

Reply:
gonna change this to be a bytes xfer counter instead.

Reply:
Do you mind adding the count of transferred files, please? I believe both are useful for debugging (I'm currently debugging a transfer which lead to WAL loss).